helsingborg-stad/aterbruket-webapp
# Feature/should not zoom

**Explain the changes you’ve made**
 Font-size a little bigger on placeholder in input at login screen

**Explain why these changes are made**
Because the font-size was too small so the app zoomed in when focus on the input, and kept that zooming when you entered the home-screen after logging in. So now, nothing zooms in.
 
**Explain your solution**
Customized the amplify authenticators default font-size on input field, from 0.875rem to 1rem.  At the root in app.css. 
 
**How to test the changes?**

1. open this link on your phone: http://testanna.d21bj89y0jdxx1.amplifyapp.com/
2. login and make sure nothing zooms in
